src/Containers/BoxComposition2/BoxComposition2.tsx

```diff
@@ -0,0 +1,115 @@
+import React,{useState,useEffect} from 'react'
+import styled from 'styled-components'
+
+interface IBoxProps{
+    imgSrc:string;
+    top:number;
+    left:number;
+    width:number;
+    height:number;
+    borderRadius:number;
+}
+
+export interface IBoxComposition2Props{
+    box1:IBoxProps;
+    box2:IBoxProps;
+    box3:IBoxProps;
+}
+
+export const BoxComposition2:React.FC<IBoxComposition2Props>=({box1,box2,box3}):React.ReactElement=>{
+    const [showBox2,setShowBox2]=useState(false)
+    const [showBox3,setShowBox3]=useState(false)
+    useEffect(()=>{
+        setTimeout(()=>{
+            setShowBox2(true)
+        },1000)
+        setTimeout(()=>{
+            setShowBox3(true)
+        },2000)
+    },[])
+    return     (
+        <div>
```

Comment:
where is the ...props spread here?

Reply:
I don't understand...

Reply:
You need to do
<div {...props} />
That way when people use this component they can add a margin to it!